src/ui/guiforcase4/ModifyInventory.java

```diff
@@ -17,7 +17,7 @@
 
   public ModifyInventory(Vehicle modifyV) {
     this.modifyV = modifyV;
-    this.dID = modifyV.getDealerId();
+    this.dID=modifyV.getDealerId();
```

Comment:
there should be spaces before and after =

Reply:
Yes, I solved this problem